requirements.txt

```diff
@@ -6,3 +6,5 @@ pytest==7.4.3
 pytest-mock==3.12.0
 requests-mock==1.11.0
 coverage==7.3.2
+scikit-learn
+gdown
```

Comment:
please pin a specific version, so that library changes do not break us. I know gdown is less likely to have breaking changes than openai, but it seems this is the pattern we have chosen for now. or if you know how to pin a major version number or another best practice that works too!

Reply:
Yeah, we can pin major (and minor) versions with `library_name>=major(.minor)`. This is usually better than absolute versions so that pip can pull patch updates and resolve library dependency issues

Reply:
I'll add a line item to the Engineering Excellence "Manage Outdated Dependencies" project to get dependabot setup for this repo so we can more easily maintain dependency versions going forward.